sw/device/lib/testing/lc_ctrl_testutils.c

```diff
@@ -0,0 +1,45 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+#include "sw/device/lib/testing/lc_ctrl_testutils.h"
+
+#include <stdbool.h>
+
+#include "sw/device/lib/dif/dif_lc_ctrl.h"
+#include "sw/device/lib/runtime/log.h"
+#include "sw/device/lib/testing/check.h"
+
+#include "hw/top_earlgrey/sw/autogen/top_earlgrey.h"
+
+lc_ctrl_t lc_ctrl_init(void) {
+  lc_ctrl_t lc;
+
+  mmio_region_t region = mmio_region_from_addr(TOP_EARLGREY_LC_CTRL_BASE_ADDR);
+  CHECK_DIF_OK(dif_lc_ctrl_init(region, &lc.dif));
+
+  return lc;
+}
+
+bool lc_ctrl_debug_func_enabled(const lc_ctrl_t *lc) {
```

Comment:
testutils functions should have the format: `<ip>_testutils_<func-name>`, so maybe just change this this function name to `lc_ctrl_testutils_debug_func_enabled(...)`?

Reply:
Done.